digraph "unitGraph" {
    "Block 0:
[preds: ] [succs: 1 7 ]
r43 = newarray (java.security.cert.Certificate)[0];
"
    "Block 1:
[preds: 0 ] [succs: 2 7 ]
$r0 = new java.security.KeyStore$PrivateKeyEntry;
"
    "Block 2:
[preds: 1 ] [succs: 3 7 ]
specialinvoke $r0.<java.security.KeyStore$PrivateKeyEntry: void <init>(java.security.PrivateKey,java.security.cert.Certificate[])>(null, r43);
"
    "Block 3:
[preds: 2 ] [succs: 4 7 ]
r44 = $r0;
"
    "Block 4:
[preds: 3 ] [succs: 5 7 ]
$r1 = new java.lang.SecurityException;
"
    "Block 5:
[preds: 4 ] [succs: 6 7 ]
specialinvoke $r1.<java.lang.SecurityException: void <init>(java.lang.String)>(\"test 1 failed\");
"
    "Block 6:
[preds: 5 ] [succs: 7 ]
throw $r1;
"
    "Block 7:
[preds: 0 1 2 3 4 5 6 ] [succs: 8 16 ]
$r38 := @caughtexception;
r45 = $r38;
$r39 = <java.lang.System: java.io.PrintStream out>;
virtualinvoke $r39.<java.io.PrintStream: void println(java.lang.String)>(\"test 1 passed\");
"
    "Block 8:
[preds: 7 ] [succs: 9 16 ]
$r40 = new java.security.KeyStore$PrivateKeyEntry;
"
    "Block 9:
[preds: 8 ] [succs: 10 16 ]
$r41 = new javaT.securityT.KeyStore.DefaultEntryType$PrivKey1;
"
    "Block 10:
[preds: 9 ] [succs: 11 16 ]
specialinvoke $r41.<javaT.securityT.KeyStore.DefaultEntryType$PrivKey1: void <init>(javaT.securityT.KeyStore.DefaultEntryType$1)>(null);
"
    "Block 11:
[preds: 10 ] [succs: 12 16 ]
specialinvoke $r40.<java.security.KeyStore$PrivateKeyEntry: void <init>(java.security.PrivateKey,java.security.cert.Certificate[])>($r41, null);
"
    "Block 12:
[preds: 11 ] [succs: 13 16 ]
r46 = $r40;
"
    "Block 13:
[preds: 12 ] [succs: 14 16 ]
$r42 = new java.lang.SecurityException;
"
    "Block 14:
[preds: 13 ] [succs: 15 16 ]
specialinvoke $r42.<java.lang.SecurityException: void <init>(java.lang.String)>(\"test 2 failed\");
"
    "Block 15:
[preds: 14 ] [succs: 16 ]
throw $r42;
"
    "Block 16:
[preds: 7 8 9 10 11 12 13 14 15 ] [succs: 17 26 ]
$r33 := @caughtexception;
r47 = $r33;
$r34 = <java.lang.System: java.io.PrintStream out>;
virtualinvoke $r34.<java.io.PrintStream: void println(java.lang.String)>(\"test 2 passed\");
"
    "Block 17:
[preds: 16 ] [succs: 18 26 ]
r48 = newarray (java.security.cert.Certificate)[0];
"
    "Block 18:
[preds: 17 ] [succs: 19 26 ]
$r35 = new java.security.KeyStore$PrivateKeyEntry;
"
    "Block 19:
[preds: 18 ] [succs: 20 26 ]
$r36 = new javaT.securityT.KeyStore.DefaultEntryType$PrivKey1;
"
    "Block 20:
[preds: 19 ] [succs: 21 26 ]
specialinvoke $r36.<javaT.securityT.KeyStore.DefaultEntryType$PrivKey1: void <init>(javaT.securityT.KeyStore.DefaultEntryType$1)>(null);
"
    "Block 21:
[preds: 20 ] [succs: 22 26 ]
specialinvoke $r35.<java.security.KeyStore$PrivateKeyEntry: void <init>(java.security.PrivateKey,java.security.cert.Certificate[])>($r36, r48);
"
    "Block 22:
[preds: 21 ] [succs: 23 26 ]
r49 = $r35;
"
    "Block 23:
[preds: 22 ] [succs: 24 26 ]
$r37 = new java.lang.SecurityException;
"
    "Block 24:
[preds: 23 ] [succs: 25 26 ]
specialinvoke $r37.<java.lang.SecurityException: void <init>(java.lang.String)>(\"test 3 failed\");
"
    "Block 25:
[preds: 24 ] [succs: 26 ]
throw $r37;
"
    "Block 26:
[preds: 16 17 18 19 20 21 22 23 24 25 ] [succs: 27 42 ]
$r26 := @caughtexception;
r50 = $r26;
$r27 = <java.lang.System: java.io.PrintStream out>;
virtualinvoke $r27.<java.io.PrintStream: void println(java.lang.String)>(\"test 3 passed\");
"
    "Block 27:
[preds: 26 ] [succs: 28 42 ]
r51 = newarray (java.security.cert.Certificate)[2];
"
    "Block 28:
[preds: 27 ] [succs: 29 42 ]
$r28 = new javaT.securityT.KeyStore.DefaultEntryType$Cert;
"
    "Block 29:
[preds: 28 ] [succs: 30 42 ]
specialinvoke $r28.<javaT.securityT.KeyStore.DefaultEntryType$Cert: void <init>()>();
"
    "Block 30:
[preds: 29 ] [succs: 31 42 ]
r51[0] = $r28;
"
    "Block 31:
[preds: 30 ] [succs: 32 42 ]
$r29 = new javaT.securityT.KeyStore.DefaultEntryType$X509Cert;
"
    "Block 32:
[preds: 31 ] [succs: 33 42 ]
specialinvoke $r29.<javaT.securityT.KeyStore.DefaultEntryType$X509Cert: void <init>(javaT.securityT.KeyStore.DefaultEntryType$1)>(null);
"
    "Block 33:
[preds: 32 ] [succs: 34 42 ]
r51[1] = $r29;
"
    "Block 34:
[preds: 33 ] [succs: 35 42 ]
$r30 = new java.security.KeyStore$PrivateKeyEntry;
"
    "Block 35:
[preds: 34 ] [succs: 36 42 ]
$r31 = new javaT.securityT.KeyStore.DefaultEntryType$PrivKey1;
"
    "Block 36:
[preds: 35 ] [succs: 37 42 ]
specialinvoke $r31.<javaT.securityT.KeyStore.DefaultEntryType$PrivKey1: void <init>(javaT.securityT.KeyStore.DefaultEntryType$1)>(null);
"
    "Block 37:
[preds: 36 ] [succs: 38 42 ]
specialinvoke $r30.<java.security.KeyStore$PrivateKeyEntry: void <init>(java.security.PrivateKey,java.security.cert.Certificate[])>($r31, r51);
"
    "Block 38:
[preds: 37 ] [succs: 39 42 ]
r52 = $r30;
"
    "Block 39:
[preds: 38 ] [succs: 40 42 ]
$r32 = new java.lang.SecurityException;
"
    "Block 40:
[preds: 39 ] [succs: 41 42 ]
specialinvoke $r32.<java.lang.SecurityException: void <init>(java.lang.String)>(\"test 4 failed\");
"
    "Block 41:
[preds: 40 ] [succs: 42 ]
throw $r32;
"
    "Block 42:
[preds: 26 27 28 29 30 31 32 33 34 35 36 37 38 39 40 41 ] [succs: 43 55 ]
$r20 := @caughtexception;
r53 = $r20;
$r21 = <java.lang.System: java.io.PrintStream out>;
virtualinvoke $r21.<java.io.PrintStream: void println(java.lang.String)>(\"test 4 passed\");
"
    "Block 43:
[preds: 42 ] [succs: 44 55 ]
r54 = newarray (java.security.cert.Certificate)[1];
"
    "Block 44:
[preds: 43 ] [succs: 45 55 ]
$r22 = new javaT.securityT.KeyStore.DefaultEntryType$Cert;
"
    "Block 45:
[preds: 44 ] [succs: 46 55 ]
specialinvoke $r22.<javaT.securityT.KeyStore.DefaultEntryType$Cert: void <init>()>();
"
    "Block 46:
[preds: 45 ] [succs: 47 55 ]
r54[0] = $r22;
"
    "Block 47:
[preds: 46 ] [succs: 48 55 ]
$r23 = new java.security.KeyStore$PrivateKeyEntry;
"
    "Block 48:
[preds: 47 ] [succs: 49 55 ]
$r24 = new javaT.securityT.KeyStore.DefaultEntryType$PrivKey1;
"
    "Block 49:
[preds: 48 ] [succs: 50 55 ]
specialinvoke $r24.<javaT.securityT.KeyStore.DefaultEntryType$PrivKey1: void <init>(javaT.securityT.KeyStore.DefaultEntryType$1)>(null);
"
    "Block 50:
[preds: 49 ] [succs: 51 55 ]
specialinvoke $r23.<java.security.KeyStore$PrivateKeyEntry: void <init>(java.security.PrivateKey,java.security.cert.Certificate[])>($r24, r54);
"
    "Block 51:
[preds: 50 ] [succs: 52 55 ]
r55 = $r23;
"
    "Block 52:
[preds: 51 ] [succs: 53 55 ]
$r25 = new java.lang.SecurityException;
"
    "Block 53:
[preds: 52 ] [succs: 54 55 ]
specialinvoke $r25.<java.lang.SecurityException: void <init>(java.lang.String)>(\"test 5 failed\");
"
    "Block 54:
[preds: 53 ] [succs: 55 ]
throw $r25;
"
    "Block 55:
[preds: 42 43 44 45 46 47 48 49 50 51 52 53 54 ] [succs: 56 57 ]
$r2 := @caughtexception;
r56 = $r2;
$r3 = <java.lang.System: java.io.PrintStream out>;
virtualinvoke $r3.<java.io.PrintStream: void println(java.lang.String)>(\"test 5 passed\");
r57 = newarray (java.security.cert.Certificate)[2];
$r4 = new javaT.securityT.KeyStore.DefaultEntryType$X509Cert;
specialinvoke $r4.<javaT.securityT.KeyStore.DefaultEntryType$X509Cert: void <init>(javaT.securityT.KeyStore.DefaultEntryType$1)>(null);
r57[0] = $r4;
$r5 = new javaT.securityT.KeyStore.DefaultEntryType$X509Cert;
specialinvoke $r5.<javaT.securityT.KeyStore.DefaultEntryType$X509Cert: void <init>(javaT.securityT.KeyStore.DefaultEntryType$1)>(null);
r57[1] = $r5;
$r6 = new javaT.securityT.KeyStore.DefaultEntryType$PrivKey1;
specialinvoke $r6.<javaT.securityT.KeyStore.DefaultEntryType$PrivKey1: void <init>(javaT.securityT.KeyStore.DefaultEntryType$1)>(null);
r58 = $r6;
$r7 = new java.security.KeyStore$PrivateKeyEntry;
specialinvoke $r7.<java.security.KeyStore$PrivateKeyEntry: void <init>(java.security.PrivateKey,java.security.cert.Certificate[])>(r58, r57);
r8 = $r7;
r9 = virtualinvoke r8.<java.security.KeyStore$PrivateKeyEntry: java.security.cert.Certificate[] getCertificateChain()>();
$z0 = r9 instanceof java.security.cert.X509Certificate[];
if $z0 == 0 goto $r10 = new java.lang.SecurityException;
"
    "Block 56:
[preds: 55 ] [succs: 58 ]
$r11 = <java.lang.System: java.io.PrintStream out>;
virtualinvoke $r11.<java.io.PrintStream: void println(java.lang.String)>(\"test 6 passed\");
goto [?= $i0 = lengthof r9];
"
    "Block 57:
[preds: 55 ] [succs: ]
$r10 = new java.lang.SecurityException;
specialinvoke $r10.<java.lang.SecurityException: void <init>(java.lang.String)>(\"test 6 failed\");
throw $r10;
"
    "Block 58:
[preds: 56 ] [succs: 59 62 ]
$i0 = lengthof r9;
if $i0 != 2 goto $r59 = new java.lang.SecurityException;
"
    "Block 59:
[preds: 58 ] [succs: 60 62 ]
$r13 = r9[0];
$r12 = r57[0];
if $r13 != $r12 goto $r59 = new java.lang.SecurityException;
"
    "Block 60:
[preds: 59 ] [succs: 61 62 ]
$r15 = r9[1];
$r14 = r57[1];
if $r15 != $r14 goto $r59 = new java.lang.SecurityException;
"
    "Block 61:
[preds: 60 ] [succs: 63 ]
$r16 = <java.lang.System: java.io.PrintStream out>;
virtualinvoke $r16.<java.io.PrintStream: void println(java.lang.String)>(\"test 7 passed\");
goto [?= $r17 = virtualinvoke r8.<java.security.KeyStore$PrivateKeyEntry: java.security.PrivateKey getPrivateKey()>()];
"
    "Block 62:
[preds: 58 59 60 ] [succs: ]
$r59 = new java.lang.SecurityException;
specialinvoke $r59.<java.lang.SecurityException: void <init>(java.lang.String)>(\"test 7 failed\");
throw $r59;
"
    "Block 63:
[preds: 61 ] [succs: 64 65 ]
$r17 = virtualinvoke r8.<java.security.KeyStore$PrivateKeyEntry: java.security.PrivateKey getPrivateKey()>();
if $r17 != r58 goto $r18 = new java.lang.SecurityException;
"
    "Block 64:
[preds: 63 ] [succs: 66 ]
$r19 = <java.lang.System: java.io.PrintStream out>;
virtualinvoke $r19.<java.io.PrintStream: void println(java.lang.String)>(\"test 8 passed\");
goto [?= return];
"
    "Block 65:
[preds: 63 ] [succs: ]
$r18 = new java.lang.SecurityException;
specialinvoke $r18.<java.lang.SecurityException: void <init>(java.lang.String)>(\"test 8 failed\");
throw $r18;
"
    "Block 66:
[preds: 64 ] [succs: ]
return;
"
    "Block 0:
[preds: ] [succs: 1 7 ]
r43 = newarray (java.security.cert.Certificate)[0];
"->"Block 1:
[preds: 0 ] [succs: 2 7 ]
$r0 = new java.security.KeyStore$PrivateKeyEntry;
";
    "Block 0:
[preds: ] [succs: 1 7 ]
r43 = newarray (java.security.cert.Certificate)[0];
"->"Block 7:
[preds: 0 1 2 3 4 5 6 ] [succs: 8 16 ]
$r38 := @caughtexception;
r45 = $r38;
$r39 = <java.lang.System: java.io.PrintStream out>;
virtualinvoke $r39.<java.io.PrintStream: void println(java.lang.String)>(\"test 1 passed\");
";
    "Block 1:
[preds: 0 ] [succs: 2 7 ]
$r0 = new java.security.KeyStore$PrivateKeyEntry;
"->"Block 2:
[preds: 1 ] [succs: 3 7 ]
specialinvoke $r0.<java.security.KeyStore$PrivateKeyEntry: void <init>(java.security.PrivateKey,java.security.cert.Certificate[])>(null, r43);
";
    "Block 1:
[preds: 0 ] [succs: 2 7 ]
$r0 = new java.security.KeyStore$PrivateKeyEntry;
"->"Block 7:
[preds: 0 1 2 3 4 5 6 ] [succs: 8 16 ]
$r38 := @caughtexception;
r45 = $r38;
$r39 = <java.lang.System: java.io.PrintStream out>;
virtualinvoke $r39.<java.io.PrintStream: void println(java.lang.String)>(\"test 1 passed\");
";
    "Block 2:
[preds: 1 ] [succs: 3 7 ]
specialinvoke $r0.<java.security.KeyStore$PrivateKeyEntry: void <init>(java.security.PrivateKey,java.security.cert.Certificate[])>(null, r43);
"->"Block 3:
[preds: 2 ] [succs: 4 7 ]
r44 = $r0;
";
    "Block 2:
[preds: 1 ] [succs: 3 7 ]
specialinvoke $r0.<java.security.KeyStore$PrivateKeyEntry: void <init>(java.security.PrivateKey,java.security.cert.Certificate[])>(null, r43);
"->"Block 7:
[preds: 0 1 2 3 4 5 6 ] [succs: 8 16 ]
$r38 := @caughtexception;
r45 = $r38;
$r39 = <java.lang.System: java.io.PrintStream out>;
virtualinvoke $r39.<java.io.PrintStream: void println(java.lang.String)>(\"test 1 passed\");
";
    "Block 3:
[preds: 2 ] [succs: 4 7 ]
r44 = $r0;
"->"Block 4:
[preds: 3 ] [succs: 5 7 ]
$r1 = new java.lang.SecurityException;
";
    "Block 3:
[preds: 2 ] [succs: 4 7 ]
r44 = $r0;
"->"Block 7:
[preds: 0 1 2 3 4 5 6 ] [succs: 8 16 ]
$r38 := @caughtexception;
r45 = $r38;
$r39 = <java.lang.System: java.io.PrintStream out>;
virtualinvoke $r39.<java.io.PrintStream: void println(java.lang.String)>(\"test 1 passed\");
";
    "Block 4:
[preds: 3 ] [succs: 5 7 ]
$r1 = new java.lang.SecurityException;
"->"Block 5:
[preds: 4 ] [succs: 6 7 ]
specialinvoke $r1.<java.lang.SecurityException: void <init>(java.lang.String)>(\"test 1 failed\");
";
    "Block 4:
[preds: 3 ] [succs: 5 7 ]
$r1 = new java.lang.SecurityException;
"->"Block 7:
[preds: 0 1 2 3 4 5 6 ] [succs: 8 16 ]
$r38 := @caughtexception;
r45 = $r38;
$r39 = <java.lang.System: java.io.PrintStream out>;
virtualinvoke $r39.<java.io.PrintStream: void println(java.lang.String)>(\"test 1 passed\");
";
    "Block 5:
[preds: 4 ] [succs: 6 7 ]
specialinvoke $r1.<java.lang.SecurityException: void <init>(java.lang.String)>(\"test 1 failed\");
"->"Block 6:
[preds: 5 ] [succs: 7 ]
throw $r1;
";
    "Block 5:
[preds: 4 ] [succs: 6 7 ]
specialinvoke $r1.<java.lang.SecurityException: void <init>(java.lang.String)>(\"test 1 failed\");
"->"Block 7:
[preds: 0 1 2 3 4 5 6 ] [succs: 8 16 ]
$r38 := @caughtexception;
r45 = $r38;
$r39 = <java.lang.System: java.io.PrintStream out>;
virtualinvoke $r39.<java.io.PrintStream: void println(java.lang.String)>(\"test 1 passed\");
";
    "Block 6:
[preds: 5 ] [succs: 7 ]
throw $r1;
"->"Block 7:
[preds: 0 1 2 3 4 5 6 ] [succs: 8 16 ]
$r38 := @caughtexception;
r45 = $r38;
$r39 = <java.lang.System: java.io.PrintStream out>;
virtualinvoke $r39.<java.io.PrintStream: void println(java.lang.String)>(\"test 1 passed\");
";
    "Block 7:
[preds: 0 1 2 3 4 5 6 ] [succs: 8 16 ]
$r38 := @caughtexception;
r45 = $r38;
$r39 = <java.lang.System: java.io.PrintStream out>;
virtualinvoke $r39.<java.io.PrintStream: void println(java.lang.String)>(\"test 1 passed\");
"->"Block 8:
[preds: 7 ] [succs: 9 16 ]
$r40 = new java.security.KeyStore$PrivateKeyEntry;
";
    "Block 7:
[preds: 0 1 2 3 4 5 6 ] [succs: 8 16 ]
$r38 := @caughtexception;
r45 = $r38;
$r39 = <java.lang.System: java.io.PrintStream out>;
virtualinvoke $r39.<java.io.PrintStream: void println(java.lang.String)>(\"test 1 passed\");
"->"Block 16:
[preds: 7 8 9 10 11 12 13 14 15 ] [succs: 17 26 ]
$r33 := @caughtexception;
r47 = $r33;
$r34 = <java.lang.System: java.io.PrintStream out>;
virtualinvoke $r34.<java.io.PrintStream: void println(java.lang.String)>(\"test 2 passed\");
";
    "Block 8:
[preds: 7 ] [succs: 9 16 ]
$r40 = new java.security.KeyStore$PrivateKeyEntry;
"->"Block 9:
[preds: 8 ] [succs: 10 16 ]
$r41 = new javaT.securityT.KeyStore.DefaultEntryType$PrivKey1;
";
    "Block 8:
[preds: 7 ] [succs: 9 16 ]
$r40 = new java.security.KeyStore$PrivateKeyEntry;
"->"Block 16:
[preds: 7 8 9 10 11 12 13 14 15 ] [succs: 17 26 ]
$r33 := @caughtexception;
r47 = $r33;
$r34 = <java.lang.System: java.io.PrintStream out>;
virtualinvoke $r34.<java.io.PrintStream: void println(java.lang.String)>(\"test 2 passed\");
";
    "Block 9:
[preds: 8 ] [succs: 10 16 ]
$r41 = new javaT.securityT.KeyStore.DefaultEntryType$PrivKey1;
"->"Block 10:
[preds: 9 ] [succs: 11 16 ]
specialinvoke $r41.<javaT.securityT.KeyStore.DefaultEntryType$PrivKey1: void <init>(javaT.securityT.KeyStore.DefaultEntryType$1)>(null);
";
    "Block 9:
[preds: 8 ] [succs: 10 16 ]
$r41 = new javaT.securityT.KeyStore.DefaultEntryType$PrivKey1;
"->"Block 16:
[preds: 7 8 9 10 11 12 13 14 15 ] [succs: 17 26 ]
$r33 := @caughtexception;
r47 = $r33;
$r34 = <java.lang.System: java.io.PrintStream out>;
virtualinvoke $r34.<java.io.PrintStream: void println(java.lang.String)>(\"test 2 passed\");
";
    "Block 10:
[preds: 9 ] [succs: 11 16 ]
specialinvoke $r41.<javaT.securityT.KeyStore.DefaultEntryType$PrivKey1: void <init>(javaT.securityT.KeyStore.DefaultEntryType$1)>(null);
"->"Block 11:
[preds: 10 ] [succs: 12 16 ]
specialinvoke $r40.<java.security.KeyStore$PrivateKeyEntry: void <init>(java.security.PrivateKey,java.security.cert.Certificate[])>($r41, null);
";
    "Block 10:
[preds: 9 ] [succs: 11 16 ]
specialinvoke $r41.<javaT.securityT.KeyStore.DefaultEntryType$PrivKey1: void <init>(javaT.securityT.KeyStore.DefaultEntryType$1)>(null);
"->"Block 16:
[preds: 7 8 9 10 11 12 13 14 15 ] [succs: 17 26 ]
$r33 := @caughtexception;
r47 = $r33;
$r34 = <java.lang.System: java.io.PrintStream out>;
virtualinvoke $r34.<java.io.PrintStream: void println(java.lang.String)>(\"test 2 passed\");
";
    "Block 11:
[preds: 10 ] [succs: 12 16 ]
specialinvoke $r40.<java.security.KeyStore$PrivateKeyEntry: void <init>(java.security.PrivateKey,java.security.cert.Certificate[])>($r41, null);
"->"Block 12:
[preds: 11 ] [succs: 13 16 ]
r46 = $r40;
";
    "Block 11:
[preds: 10 ] [succs: 12 16 ]
specialinvoke $r40.<java.security.KeyStore$PrivateKeyEntry: void <init>(java.security.PrivateKey,java.security.cert.Certificate[])>($r41, null);
"->"Block 16:
[preds: 7 8 9 10 11 12 13 14 15 ] [succs: 17 26 ]
$r33 := @caughtexception;
r47 = $r33;
$r34 = <java.lang.System: java.io.PrintStream out>;
virtualinvoke $r34.<java.io.PrintStream: void println(java.lang.String)>(\"test 2 passed\");
";
    "Block 12:
[preds: 11 ] [succs: 13 16 ]
r46 = $r40;
"->"Block 13:
[preds: 12 ] [succs: 14 16 ]
$r42 = new java.lang.SecurityException;
";
    "Block 12:
[preds: 11 ] [succs: 13 16 ]
r46 = $r40;
"->"Block 16:
[preds: 7 8 9 10 11 12 13 14 15 ] [succs: 17 26 ]
$r33 := @caughtexception;
r47 = $r33;
$r34 = <java.lang.System: java.io.PrintStream out>;
virtualinvoke $r34.<java.io.PrintStream: void println(java.lang.String)>(\"test 2 passed\");
";
    "Block 13:
[preds: 12 ] [succs: 14 16 ]
$r42 = new java.lang.SecurityException;
"->"Block 14:
[preds: 13 ] [succs: 15 16 ]
specialinvoke $r42.<java.lang.SecurityException: void <init>(java.lang.String)>(\"test 2 failed\");
";
    "Block 13:
[preds: 12 ] [succs: 14 16 ]
$r42 = new java.lang.SecurityException;
"->"Block 16:
[preds: 7 8 9 10 11 12 13 14 15 ] [succs: 17 26 ]
$r33 := @caughtexception;
r47 = $r33;
$r34 = <java.lang.System: java.io.PrintStream out>;
virtualinvoke $r34.<java.io.PrintStream: void println(java.lang.String)>(\"test 2 passed\");
";
    "Block 14:
[preds: 13 ] [succs: 15 16 ]
specialinvoke $r42.<java.lang.SecurityException: void <init>(java.lang.String)>(\"test 2 failed\");
"->"Block 15:
[preds: 14 ] [succs: 16 ]
throw $r42;
";
    "Block 14:
[preds: 13 ] [succs: 15 16 ]
specialinvoke $r42.<java.lang.SecurityException: void <init>(java.lang.String)>(\"test 2 failed\");
"->"Block 16:
[preds: 7 8 9 10 11 12 13 14 15 ] [succs: 17 26 ]
$r33 := @caughtexception;
r47 = $r33;
$r34 = <java.lang.System: java.io.PrintStream out>;
virtualinvoke $r34.<java.io.PrintStream: void println(java.lang.String)>(\"test 2 passed\");
";
    "Block 15:
[preds: 14 ] [succs: 16 ]
throw $r42;
"->"Block 16:
[preds: 7 8 9 10 11 12 13 14 15 ] [succs: 17 26 ]
$r33 := @caughtexception;
r47 = $r33;
$r34 = <java.lang.System: java.io.PrintStream out>;
virtualinvoke $r34.<java.io.PrintStream: void println(java.lang.String)>(\"test 2 passed\");
";
    "Block 16:
[preds: 7 8 9 10 11 12 13 14 15 ] [succs: 17 26 ]
$r33 := @caughtexception;
r47 = $r33;
$r34 = <java.lang.System: java.io.PrintStream out>;
virtualinvoke $r34.<java.io.PrintStream: void println(java.lang.String)>(\"test 2 passed\");
"->"Block 17:
[preds: 16 ] [succs: 18 26 ]
r48 = newarray (java.security.cert.Certificate)[0];
";
    "Block 16:
[preds: 7 8 9 10 11 12 13 14 15 ] [succs: 17 26 ]
$r33 := @caughtexception;
r47 = $r33;
$r34 = <java.lang.System: java.io.PrintStream out>;
virtualinvoke $r34.<java.io.PrintStream: void println(java.lang.String)>(\"test 2 passed\");
"->"Block 26:
[preds: 16 17 18 19 20 21 22 23 24 25 ] [succs: 27 42 ]
$r26 := @caughtexception;
r50 = $r26;
$r27 = <java.lang.System: java.io.PrintStream out>;
virtualinvoke $r27.<java.io.PrintStream: void println(java.lang.String)>(\"test 3 passed\");
";
    "Block 17:
[preds: 16 ] [succs: 18 26 ]
r48 = newarray (java.security.cert.Certificate)[0];
"->"Block 18:
[preds: 17 ] [succs: 19 26 ]
$r35 = new java.security.KeyStore$PrivateKeyEntry;
";
    "Block 17:
[preds: 16 ] [succs: 18 26 ]
r48 = newarray (java.security.cert.Certificate)[0];
"->"Block 26:
[preds: 16 17 18 19 20 21 22 23 24 25 ] [succs: 27 42 ]
$r26 := @caughtexception;
r50 = $r26;
$r27 = <java.lang.System: java.io.PrintStream out>;
virtualinvoke $r27.<java.io.PrintStream: void println(java.lang.String)>(\"test 3 passed\");
";
    "Block 18:
[preds: 17 ] [succs: 19 26 ]
$r35 = new java.security.KeyStore$PrivateKeyEntry;
"->"Block 19:
[preds: 18 ] [succs: 20 26 ]
$r36 = new javaT.securityT.KeyStore.DefaultEntryType$PrivKey1;
";
    "Block 18:
[preds: 17 ] [succs: 19 26 ]
$r35 = new java.security.KeyStore$PrivateKeyEntry;
"->"Block 26:
[preds: 16 17 18 19 20 21 22 23 24 25 ] [succs: 27 42 ]
$r26 := @caughtexception;
r50 = $r26;
$r27 = <java.lang.System: java.io.PrintStream out>;
virtualinvoke $r27.<java.io.PrintStream: void println(java.lang.String)>(\"test 3 passed\");
";
    "Block 19:
[preds: 18 ] [succs: 20 26 ]
$r36 = new javaT.securityT.KeyStore.DefaultEntryType$PrivKey1;
"->"Block 20:
[preds: 19 ] [succs: 21 26 ]
specialinvoke $r36.<javaT.securityT.KeyStore.DefaultEntryType$PrivKey1: void <init>(javaT.securityT.KeyStore.DefaultEntryType$1)>(null);
";
    "Block 19:
[preds: 18 ] [succs: 20 26 ]
$r36 = new javaT.securityT.KeyStore.DefaultEntryType$PrivKey1;
"->"Block 26:
[preds: 16 17 18 19 20 21 22 23 24 25 ] [succs: 27 42 ]
$r26 := @caughtexception;
r50 = $r26;
$r27 = <java.lang.System: java.io.PrintStream out>;
virtualinvoke $r27.<java.io.PrintStream: void println(java.lang.String)>(\"test 3 passed\");
";
    "Block 20:
[preds: 19 ] [succs: 21 26 ]
specialinvoke $r36.<javaT.securityT.KeyStore.DefaultEntryType$PrivKey1: void <init>(javaT.securityT.KeyStore.DefaultEntryType$1)>(null);
"->"Block 21:
[preds: 20 ] [succs: 22 26 ]
specialinvoke $r35.<java.security.KeyStore$PrivateKeyEntry: void <init>(java.security.PrivateKey,java.security.cert.Certificate[])>($r36, r48);
";
    "Block 20:
[preds: 19 ] [succs: 21 26 ]
specialinvoke $r36.<javaT.securityT.KeyStore.DefaultEntryType$PrivKey1: void <init>(javaT.securityT.KeyStore.DefaultEntryType$1)>(null);
"->"Block 26:
[preds: 16 17 18 19 20 21 22 23 24 25 ] [succs: 27 42 ]
$r26 := @caughtexception;
r50 = $r26;
$r27 = <java.lang.System: java.io.PrintStream out>;
virtualinvoke $r27.<java.io.PrintStream: void println(java.lang.String)>(\"test 3 passed\");
";
    "Block 21:
[preds: 20 ] [succs: 22 26 ]
specialinvoke $r35.<java.security.KeyStore$PrivateKeyEntry: void <init>(java.security.PrivateKey,java.security.cert.Certificate[])>($r36, r48);
"->"Block 22:
[preds: 21 ] [succs: 23 26 ]
r49 = $r35;
";
    "Block 21:
[preds: 20 ] [succs: 22 26 ]
specialinvoke $r35.<java.security.KeyStore$PrivateKeyEntry: void <init>(java.security.PrivateKey,java.security.cert.Certificate[])>($r36, r48);
"->"Block 26:
[preds: 16 17 18 19 20 21 22 23 24 25 ] [succs: 27 42 ]
$r26 := @caughtexception;
r50 = $r26;
$r27 = <java.lang.System: java.io.PrintStream out>;
virtualinvoke $r27.<java.io.PrintStream: void println(java.lang.String)>(\"test 3 passed\");
";
    "Block 22:
[preds: 21 ] [succs: 23 26 ]
r49 = $r35;
"->"Block 23:
[preds: 22 ] [succs: 24 26 ]
$r37 = new java.lang.SecurityException;
";
    "Block 22:
[preds: 21 ] [succs: 23 26 ]
r49 = $r35;
"->"Block 26:
[preds: 16 17 18 19 20 21 22 23 24 25 ] [succs: 27 42 ]
$r26 := @caughtexception;
r50 = $r26;
$r27 = <java.lang.System: java.io.PrintStream out>;
virtualinvoke $r27.<java.io.PrintStream: void println(java.lang.String)>(\"test 3 passed\");
";
    "Block 23:
[preds: 22 ] [succs: 24 26 ]
$r37 = new java.lang.SecurityException;
"->"Block 24:
[preds: 23 ] [succs: 25 26 ]
specialinvoke $r37.<java.lang.SecurityException: void <init>(java.lang.String)>(\"test 3 failed\");
";
    "Block 23:
[preds: 22 ] [succs: 24 26 ]
$r37 = new java.lang.SecurityException;
"->"Block 26:
[preds: 16 17 18 19 20 21 22 23 24 25 ] [succs: 27 42 ]
$r26 := @caughtexception;
r50 = $r26;
$r27 = <java.lang.System: java.io.PrintStream out>;
virtualinvoke $r27.<java.io.PrintStream: void println(java.lang.String)>(\"test 3 passed\");
";
    "Block 24:
[preds: 23 ] [succs: 25 26 ]
specialinvoke $r37.<java.lang.SecurityException: void <init>(java.lang.String)>(\"test 3 failed\");
"->"Block 25:
[preds: 24 ] [succs: 26 ]
throw $r37;
";
    "Block 24:
[preds: 23 ] [succs: 25 26 ]
specialinvoke $r37.<java.lang.SecurityException: void <init>(java.lang.String)>(\"test 3 failed\");
"->"Block 26:
[preds: 16 17 18 19 20 21 22 23 24 25 ] [succs: 27 42 ]
$r26 := @caughtexception;
r50 = $r26;
$r27 = <java.lang.System: java.io.PrintStream out>;
virtualinvoke $r27.<java.io.PrintStream: void println(java.lang.String)>(\"test 3 passed\");
";
    "Block 25:
[preds: 24 ] [succs: 26 ]
throw $r37;
"->"Block 26:
[preds: 16 17 18 19 20 21 22 23 24 25 ] [succs: 27 42 ]
$r26 := @caughtexception;
r50 = $r26;
$r27 = <java.lang.System: java.io.PrintStream out>;
virtualinvoke $r27.<java.io.PrintStream: void println(java.lang.String)>(\"test 3 passed\");
";
    "Block 26:
[preds: 16 17 18 19 20 21 22 23 24 25 ] [succs: 27 42 ]
$r26 := @caughtexception;
r50 = $r26;
$r27 = <java.lang.System: java.io.PrintStream out>;
virtualinvoke $r27.<java.io.PrintStream: void println(java.lang.String)>(\"test 3 passed\");
"->"Block 27:
[preds: 26 ] [succs: 28 42 ]
r51 = newarray (java.security.cert.Certificate)[2];
";
    "Block 26:
[preds: 16 17 18 19 20 21 22 23 24 25 ] [succs: 27 42 ]
$r26 := @caughtexception;
r50 = $r26;
$r27 = <java.lang.System: java.io.PrintStream out>;
virtualinvoke $r27.<java.io.PrintStream: void println(java.lang.String)>(\"test 3 passed\");
"->"Block 42:
[preds: 26 27 28 29 30 31 32 33 34 35 36 37 38 39 40 41 ] [succs: 43 55 ]
$r20 := @caughtexception;
r53 = $r20;
$r21 = <java.lang.System: java.io.PrintStream out>;
virtualinvoke $r21.<java.io.PrintStream: void println(java.lang.String)>(\"test 4 passed\");
";
    "Block 27:
[preds: 26 ] [succs: 28 42 ]
r51 = newarray (java.security.cert.Certificate)[2];
"->"Block 28:
[preds: 27 ] [succs: 29 42 ]
$r28 = new javaT.securityT.KeyStore.DefaultEntryType$Cert;
";
    "Block 27:
[preds: 26 ] [succs: 28 42 ]
r51 = newarray (java.security.cert.Certificate)[2];
"->"Block 42:
[preds: 26 27 28 29 30 31 32 33 34 35 36 37 38 39 40 41 ] [succs: 43 55 ]
$r20 := @caughtexception;
r53 = $r20;
$r21 = <java.lang.System: java.io.PrintStream out>;
virtualinvoke $r21.<java.io.PrintStream: void println(java.lang.String)>(\"test 4 passed\");
";
    "Block 28:
[preds: 27 ] [succs: 29 42 ]
$r28 = new javaT.securityT.KeyStore.DefaultEntryType$Cert;
"->"Block 29:
[preds: 28 ] [succs: 30 42 ]
specialinvoke $r28.<javaT.securityT.KeyStore.DefaultEntryType$Cert: void <init>()>();
";
    "Block 28:
[preds: 27 ] [succs: 29 42 ]
$r28 = new javaT.securityT.KeyStore.DefaultEntryType$Cert;
"->"Block 42:
[preds: 26 27 28 29 30 31 32 33 34 35 36 37 38 39 40 41 ] [succs: 43 55 ]
$r20 := @caughtexception;
r53 = $r20;
$r21 = <java.lang.System: java.io.PrintStream out>;
virtualinvoke $r21.<java.io.PrintStream: void println(java.lang.String)>(\"test 4 passed\");
";
    "Block 29:
[preds: 28 ] [succs: 30 42 ]
specialinvoke $r28.<javaT.securityT.KeyStore.DefaultEntryType$Cert: void <init>()>();
"->"Block 30:
[preds: 29 ] [succs: 31 42 ]
r51[0] = $r28;
";
    "Block 29:
[preds: 28 ] [succs: 30 42 ]
specialinvoke $r28.<javaT.securityT.KeyStore.DefaultEntryType$Cert: void <init>()>();
"->"Block 42:
[preds: 26 27 28 29 30 31 32 33 34 35 36 37 38 39 40 41 ] [succs: 43 55 ]
$r20 := @caughtexception;
r53 = $r20;
$r21 = <java.lang.System: java.io.PrintStream out>;
virtualinvoke $r21.<java.io.PrintStream: void println(java.lang.String)>(\"test 4 passed\");
";
    "Block 30:
[preds: 29 ] [succs: 31 42 ]
r51[0] = $r28;
"->"Block 31:
[preds: 30 ] [succs: 32 42 ]
$r29 = new javaT.securityT.KeyStore.DefaultEntryType$X509Cert;
";
    "Block 30:
[preds: 29 ] [succs: 31 42 ]
r51[0] = $r28;
"->"Block 42:
[preds: 26 27 28 29 30 31 32 33 34 35 36 37 38 39 40 41 ] [succs: 43 55 ]
$r20 := @caughtexception;
r53 = $r20;
$r21 = <java.lang.System: java.io.PrintStream out>;
virtualinvoke $r21.<java.io.PrintStream: void println(java.lang.String)>(\"test 4 passed\");
";
    "Block 31:
[preds: 30 ] [succs: 32 42 ]
$r29 = new javaT.securityT.KeyStore.DefaultEntryType$X509Cert;
"->"Block 32:
[preds: 31 ] [succs: 33 42 ]
specialinvoke $r29.<javaT.securityT.KeyStore.DefaultEntryType$X509Cert: void <init>(javaT.securityT.KeyStore.DefaultEntryType$1)>(null);
";
    "Block 31:
[preds: 30 ] [succs: 32 42 ]
$r29 = new javaT.securityT.KeyStore.DefaultEntryType$X509Cert;
"->"Block 42:
[preds: 26 27 28 29 30 31 32 33 34 35 36 37 38 39 40 41 ] [succs: 43 55 ]
$r20 := @caughtexception;
r53 = $r20;
$r21 = <java.lang.System: java.io.PrintStream out>;
virtualinvoke $r21.<java.io.PrintStream: void println(java.lang.String)>(\"test 4 passed\");
";
    "Block 32:
[preds: 31 ] [succs: 33 42 ]
specialinvoke $r29.<javaT.securityT.KeyStore.DefaultEntryType$X509Cert: void <init>(javaT.securityT.KeyStore.DefaultEntryType$1)>(null);
"->"Block 33:
[preds: 32 ] [succs: 34 42 ]
r51[1] = $r29;
";
    "Block 32:
[preds: 31 ] [succs: 33 42 ]
specialinvoke $r29.<javaT.securityT.KeyStore.DefaultEntryType$X509Cert: void <init>(javaT.securityT.KeyStore.DefaultEntryType$1)>(null);
"->"Block 42:
[preds: 26 27 28 29 30 31 32 33 34 35 36 37 38 39 40 41 ] [succs: 43 55 ]
$r20 := @caughtexception;
r53 = $r20;
$r21 = <java.lang.System: java.io.PrintStream out>;
virtualinvoke $r21.<java.io.PrintStream: void println(java.lang.String)>(\"test 4 passed\");
";
    "Block 33:
[preds: 32 ] [succs: 34 42 ]
r51[1] = $r29;
"->"Block 34:
[preds: 33 ] [succs: 35 42 ]
$r30 = new java.security.KeyStore$PrivateKeyEntry;
";
    "Block 33:
[preds: 32 ] [succs: 34 42 ]
r51[1] = $r29;
"->"Block 42:
[preds: 26 27 28 29 30 31 32 33 34 35 36 37 38 39 40 41 ] [succs: 43 55 ]
$r20 := @caughtexception;
r53 = $r20;
$r21 = <java.lang.System: java.io.PrintStream out>;
virtualinvoke $r21.<java.io.PrintStream: void println(java.lang.String)>(\"test 4 passed\");
";
    "Block 34:
[preds: 33 ] [succs: 35 42 ]
$r30 = new java.security.KeyStore$PrivateKeyEntry;
"->"Block 35:
[preds: 34 ] [succs: 36 42 ]
$r31 = new javaT.securityT.KeyStore.DefaultEntryType$PrivKey1;
";
    "Block 34:
[preds: 33 ] [succs: 35 42 ]
$r30 = new java.security.KeyStore$PrivateKeyEntry;
"->"Block 42:
[preds: 26 27 28 29 30 31 32 33 34 35 36 37 38 39 40 41 ] [succs: 43 55 ]
$r20 := @caughtexception;
r53 = $r20;
$r21 = <java.lang.System: java.io.PrintStream out>;
virtualinvoke $r21.<java.io.PrintStream: void println(java.lang.String)>(\"test 4 passed\");
";
    "Block 35:
[preds: 34 ] [succs: 36 42 ]
$r31 = new javaT.securityT.KeyStore.DefaultEntryType$PrivKey1;
"->"Block 36:
[preds: 35 ] [succs: 37 42 ]
specialinvoke $r31.<javaT.securityT.KeyStore.DefaultEntryType$PrivKey1: void <init>(javaT.securityT.KeyStore.DefaultEntryType$1)>(null);
";
    "Block 35:
[preds: 34 ] [succs: 36 42 ]
$r31 = new javaT.securityT.KeyStore.DefaultEntryType$PrivKey1;
"->"Block 42:
[preds: 26 27 28 29 30 31 32 33 34 35 36 37 38 39 40 41 ] [succs: 43 55 ]
$r20 := @caughtexception;
r53 = $r20;
$r21 = <java.lang.System: java.io.PrintStream out>;
virtualinvoke $r21.<java.io.PrintStream: void println(java.lang.String)>(\"test 4 passed\");
";
    "Block 36:
[preds: 35 ] [succs: 37 42 ]
specialinvoke $r31.<javaT.securityT.KeyStore.DefaultEntryType$PrivKey1: void <init>(javaT.securityT.KeyStore.DefaultEntryType$1)>(null);
"->"Block 37:
[preds: 36 ] [succs: 38 42 ]
specialinvoke $r30.<java.security.KeyStore$PrivateKeyEntry: void <init>(java.security.PrivateKey,java.security.cert.Certificate[])>($r31, r51);
";
    "Block 36:
[preds: 35 ] [succs: 37 42 ]
specialinvoke $r31.<javaT.securityT.KeyStore.DefaultEntryType$PrivKey1: void <init>(javaT.securityT.KeyStore.DefaultEntryType$1)>(null);
"->"Block 42:
[preds: 26 27 28 29 30 31 32 33 34 35 36 37 38 39 40 41 ] [succs: 43 55 ]
$r20 := @caughtexception;
r53 = $r20;
$r21 = <java.lang.System: java.io.PrintStream out>;
virtualinvoke $r21.<java.io.PrintStream: void println(java.lang.String)>(\"test 4 passed\");
";
    "Block 37:
[preds: 36 ] [succs: 38 42 ]
specialinvoke $r30.<java.security.KeyStore$PrivateKeyEntry: void <init>(java.security.PrivateKey,java.security.cert.Certificate[])>($r31, r51);
"->"Block 38:
[preds: 37 ] [succs: 39 42 ]
r52 = $r30;
";
    "Block 37:
[preds: 36 ] [succs: 38 42 ]
specialinvoke $r30.<java.security.KeyStore$PrivateKeyEntry: void <init>(java.security.PrivateKey,java.security.cert.Certificate[])>($r31, r51);
"->"Block 42:
[preds: 26 27 28 29 30 31 32 33 34 35 36 37 38 39 40 41 ] [succs: 43 55 ]
$r20 := @caughtexception;
r53 = $r20;
$r21 = <java.lang.System: java.io.PrintStream out>;
virtualinvoke $r21.<java.io.PrintStream: void println(java.lang.String)>(\"test 4 passed\");
";
    "Block 38:
[preds: 37 ] [succs: 39 42 ]
r52 = $r30;
"->"Block 39:
[preds: 38 ] [succs: 40 42 ]
$r32 = new java.lang.SecurityException;
";
    "Block 38:
[preds: 37 ] [succs: 39 42 ]
r52 = $r30;
"->"Block 42:
[preds: 26 27 28 29 30 31 32 33 34 35 36 37 38 39 40 41 ] [succs: 43 55 ]
$r20 := @caughtexception;
r53 = $r20;
$r21 = <java.lang.System: java.io.PrintStream out>;
virtualinvoke $r21.<java.io.PrintStream: void println(java.lang.String)>(\"test 4 passed\");
";
    "Block 39:
[preds: 38 ] [succs: 40 42 ]
$r32 = new java.lang.SecurityException;
"->"Block 40:
[preds: 39 ] [succs: 41 42 ]
specialinvoke $r32.<java.lang.SecurityException: void <init>(java.lang.String)>(\"test 4 failed\");
";
    "Block 39:
[preds: 38 ] [succs: 40 42 ]
$r32 = new java.lang.SecurityException;
"->"Block 42:
[preds: 26 27 28 29 30 31 32 33 34 35 36 37 38 39 40 41 ] [succs: 43 55 ]
$r20 := @caughtexception;
r53 = $r20;
$r21 = <java.lang.System: java.io.PrintStream out>;
virtualinvoke $r21.<java.io.PrintStream: void println(java.lang.String)>(\"test 4 passed\");
";
    "Block 40:
[preds: 39 ] [succs: 41 42 ]
specialinvoke $r32.<java.lang.SecurityException: void <init>(java.lang.String)>(\"test 4 failed\");
"->"Block 41:
[preds: 40 ] [succs: 42 ]
throw $r32;
";
    "Block 40:
[preds: 39 ] [succs: 41 42 ]
specialinvoke $r32.<java.lang.SecurityException: void <init>(java.lang.String)>(\"test 4 failed\");
"->"Block 42:
[preds: 26 27 28 29 30 31 32 33 34 35 36 37 38 39 40 41 ] [succs: 43 55 ]
$r20 := @caughtexception;
r53 = $r20;
$r21 = <java.lang.System: java.io.PrintStream out>;
virtualinvoke $r21.<java.io.PrintStream: void println(java.lang.String)>(\"test 4 passed\");
";
    "Block 41:
[preds: 40 ] [succs: 42 ]
throw $r32;
"->"Block 42:
[preds: 26 27 28 29 30 31 32 33 34 35 36 37 38 39 40 41 ] [succs: 43 55 ]
$r20 := @caughtexception;
r53 = $r20;
$r21 = <java.lang.System: java.io.PrintStream out>;
virtualinvoke $r21.<java.io.PrintStream: void println(java.lang.String)>(\"test 4 passed\");
";
    "Block 42:
[preds: 26 27 28 29 30 31 32 33 34 35 36 37 38 39 40 41 ] [succs: 43 55 ]
$r20 := @caughtexception;
r53 = $r20;
$r21 = <java.lang.System: java.io.PrintStream out>;
virtualinvoke $r21.<java.io.PrintStream: void println(java.lang.String)>(\"test 4 passed\");
"->"Block 43:
[preds: 42 ] [succs: 44 55 ]
r54 = newarray (java.security.cert.Certificate)[1];
";
    "Block 42:
[preds: 26 27 28 29 30 31 32 33 34 35 36 37 38 39 40 41 ] [succs: 43 55 ]
$r20 := @caughtexception;
r53 = $r20;
$r21 = <java.lang.System: java.io.PrintStream out>;
virtualinvoke $r21.<java.io.PrintStream: void println(java.lang.String)>(\"test 4 passed\");
"->"Block 55:
[preds: 42 43 44 45 46 47 48 49 50 51 52 53 54 ] [succs: 56 57 ]
$r2 := @caughtexception;
r56 = $r2;
$r3 = <java.lang.System: java.io.PrintStream out>;
virtualinvoke $r3.<java.io.PrintStream: void println(java.lang.String)>(\"test 5 passed\");
r57 = newarray (java.security.cert.Certificate)[2];
$r4 = new javaT.securityT.KeyStore.DefaultEntryType$X509Cert;
specialinvoke $r4.<javaT.securityT.KeyStore.DefaultEntryType$X509Cert: void <init>(javaT.securityT.KeyStore.DefaultEntryType$1)>(null);
r57[0] = $r4;
$r5 = new javaT.securityT.KeyStore.DefaultEntryType$X509Cert;
specialinvoke $r5.<javaT.securityT.KeyStore.DefaultEntryType$X509Cert: void <init>(javaT.securityT.KeyStore.DefaultEntryType$1)>(null);
r57[1] = $r5;
$r6 = new javaT.securityT.KeyStore.DefaultEntryType$PrivKey1;
specialinvoke $r6.<javaT.securityT.KeyStore.DefaultEntryType$PrivKey1: void <init>(javaT.securityT.KeyStore.DefaultEntryType$1)>(null);
r58 = $r6;
$r7 = new java.security.KeyStore$PrivateKeyEntry;
specialinvoke $r7.<java.security.KeyStore$PrivateKeyEntry: void <init>(java.security.PrivateKey,java.security.cert.Certificate[])>(r58, r57);
r8 = $r7;
r9 = virtualinvoke r8.<java.security.KeyStore$PrivateKeyEntry: java.security.cert.Certificate[] getCertificateChain()>();
$z0 = r9 instanceof java.security.cert.X509Certificate[];
if $z0 == 0 goto $r10 = new java.lang.SecurityException;
";
    "Block 43:
[preds: 42 ] [succs: 44 55 ]
r54 = newarray (java.security.cert.Certificate)[1];
"->"Block 44:
[preds: 43 ] [succs: 45 55 ]
$r22 = new javaT.securityT.KeyStore.DefaultEntryType$Cert;
";
    "Block 43:
[preds: 42 ] [succs: 44 55 ]
r54 = newarray (java.security.cert.Certificate)[1];
"->"Block 55:
[preds: 42 43 44 45 46 47 48 49 50 51 52 53 54 ] [succs: 56 57 ]
$r2 := @caughtexception;
r56 = $r2;
$r3 = <java.lang.System: java.io.PrintStream out>;
virtualinvoke $r3.<java.io.PrintStream: void println(java.lang.String)>(\"test 5 passed\");
r57 = newarray (java.security.cert.Certificate)[2];
$r4 = new javaT.securityT.KeyStore.DefaultEntryType$X509Cert;
specialinvoke $r4.<javaT.securityT.KeyStore.DefaultEntryType$X509Cert: void <init>(javaT.securityT.KeyStore.DefaultEntryType$1)>(null);
r57[0] = $r4;
$r5 = new javaT.securityT.KeyStore.DefaultEntryType$X509Cert;
specialinvoke $r5.<javaT.securityT.KeyStore.DefaultEntryType$X509Cert: void <init>(javaT.securityT.KeyStore.DefaultEntryType$1)>(null);
r57[1] = $r5;
$r6 = new javaT.securityT.KeyStore.DefaultEntryType$PrivKey1;
specialinvoke $r6.<javaT.securityT.KeyStore.DefaultEntryType$PrivKey1: void <init>(javaT.securityT.KeyStore.DefaultEntryType$1)>(null);
r58 = $r6;
$r7 = new java.security.KeyStore$PrivateKeyEntry;
specialinvoke $r7.<java.security.KeyStore$PrivateKeyEntry: void <init>(java.security.PrivateKey,java.security.cert.Certificate[])>(r58, r57);
r8 = $r7;
r9 = virtualinvoke r8.<java.security.KeyStore$PrivateKeyEntry: java.security.cert.Certificate[] getCertificateChain()>();
$z0 = r9 instanceof java.security.cert.X509Certificate[];
if $z0 == 0 goto $r10 = new java.lang.SecurityException;
";
    "Block 44:
[preds: 43 ] [succs: 45 55 ]
$r22 = new javaT.securityT.KeyStore.DefaultEntryType$Cert;
"->"Block 45:
[preds: 44 ] [succs: 46 55 ]
specialinvoke $r22.<javaT.securityT.KeyStore.DefaultEntryType$Cert: void <init>()>();
";
    "Block 44:
[preds: 43 ] [succs: 45 55 ]
$r22 = new javaT.securityT.KeyStore.DefaultEntryType$Cert;
"->"Block 55:
[preds: 42 43 44 45 46 47 48 49 50 51 52 53 54 ] [succs: 56 57 ]
$r2 := @caughtexception;
r56 = $r2;
$r3 = <java.lang.System: java.io.PrintStream out>;
virtualinvoke $r3.<java.io.PrintStream: void println(java.lang.String)>(\"test 5 passed\");
r57 = newarray (java.security.cert.Certificate)[2];
$r4 = new javaT.securityT.KeyStore.DefaultEntryType$X509Cert;
specialinvoke $r4.<javaT.securityT.KeyStore.DefaultEntryType$X509Cert: void <init>(javaT.securityT.KeyStore.DefaultEntryType$1)>(null);
r57[0] = $r4;
$r5 = new javaT.securityT.KeyStore.DefaultEntryType$X509Cert;
specialinvoke $r5.<javaT.securityT.KeyStore.DefaultEntryType$X509Cert: void <init>(javaT.securityT.KeyStore.DefaultEntryType$1)>(null);
r57[1] = $r5;
$r6 = new javaT.securityT.KeyStore.DefaultEntryType$PrivKey1;
specialinvoke $r6.<javaT.securityT.KeyStore.DefaultEntryType$PrivKey1: void <init>(javaT.securityT.KeyStore.DefaultEntryType$1)>(null);
r58 = $r6;
$r7 = new java.security.KeyStore$PrivateKeyEntry;
specialinvoke $r7.<java.security.KeyStore$PrivateKeyEntry: void <init>(java.security.PrivateKey,java.security.cert.Certificate[])>(r58, r57);
r8 = $r7;
r9 = virtualinvoke r8.<java.security.KeyStore$PrivateKeyEntry: java.security.cert.Certificate[] getCertificateChain()>();
$z0 = r9 instanceof java.security.cert.X509Certificate[];
if $z0 == 0 goto $r10 = new java.lang.SecurityException;
";
    "Block 45:
[preds: 44 ] [succs: 46 55 ]
specialinvoke $r22.<javaT.securityT.KeyStore.DefaultEntryType$Cert: void <init>()>();
"->"Block 46:
[preds: 45 ] [succs: 47 55 ]
r54[0] = $r22;
";
    "Block 45:
[preds: 44 ] [succs: 46 55 ]
specialinvoke $r22.<javaT.securityT.KeyStore.DefaultEntryType$Cert: void <init>()>();
"->"Block 55:
[preds: 42 43 44 45 46 47 48 49 50 51 52 53 54 ] [succs: 56 57 ]
$r2 := @caughtexception;
r56 = $r2;
$r3 = <java.lang.System: java.io.PrintStream out>;
virtualinvoke $r3.<java.io.PrintStream: void println(java.lang.String)>(\"test 5 passed\");
r57 = newarray (java.security.cert.Certificate)[2];
$r4 = new javaT.securityT.KeyStore.DefaultEntryType$X509Cert;
specialinvoke $r4.<javaT.securityT.KeyStore.DefaultEntryType$X509Cert: void <init>(javaT.securityT.KeyStore.DefaultEntryType$1)>(null);
r57[0] = $r4;
$r5 = new javaT.securityT.KeyStore.DefaultEntryType$X509Cert;
specialinvoke $r5.<javaT.securityT.KeyStore.DefaultEntryType$X509Cert: void <init>(javaT.securityT.KeyStore.DefaultEntryType$1)>(null);
r57[1] = $r5;
$r6 = new javaT.securityT.KeyStore.DefaultEntryType$PrivKey1;
specialinvoke $r6.<javaT.securityT.KeyStore.DefaultEntryType$PrivKey1: void <init>(javaT.securityT.KeyStore.DefaultEntryType$1)>(null);
r58 = $r6;
$r7 = new java.security.KeyStore$PrivateKeyEntry;
specialinvoke $r7.<java.security.KeyStore$PrivateKeyEntry: void <init>(java.security.PrivateKey,java.security.cert.Certificate[])>(r58, r57);
r8 = $r7;
r9 = virtualinvoke r8.<java.security.KeyStore$PrivateKeyEntry: java.security.cert.Certificate[] getCertificateChain()>();
$z0 = r9 instanceof java.security.cert.X509Certificate[];
if $z0 == 0 goto $r10 = new java.lang.SecurityException;
";
    "Block 46:
[preds: 45 ] [succs: 47 55 ]
r54[0] = $r22;
"->"Block 47:
[preds: 46 ] [succs: 48 55 ]
$r23 = new java.security.KeyStore$PrivateKeyEntry;
";
    "Block 46:
[preds: 45 ] [succs: 47 55 ]
r54[0] = $r22;
"->"Block 55:
[preds: 42 43 44 45 46 47 48 49 50 51 52 53 54 ] [succs: 56 57 ]
$r2 := @caughtexception;
r56 = $r2;
$r3 = <java.lang.System: java.io.PrintStream out>;
virtualinvoke $r3.<java.io.PrintStream: void println(java.lang.String)>(\"test 5 passed\");
r57 = newarray (java.security.cert.Certificate)[2];
$r4 = new javaT.securityT.KeyStore.DefaultEntryType$X509Cert;
specialinvoke $r4.<javaT.securityT.KeyStore.DefaultEntryType$X509Cert: void <init>(javaT.securityT.KeyStore.DefaultEntryType$1)>(null);
r57[0] = $r4;
$r5 = new javaT.securityT.KeyStore.DefaultEntryType$X509Cert;
specialinvoke $r5.<javaT.securityT.KeyStore.DefaultEntryType$X509Cert: void <init>(javaT.securityT.KeyStore.DefaultEntryType$1)>(null);
r57[1] = $r5;
$r6 = new javaT.securityT.KeyStore.DefaultEntryType$PrivKey1;
specialinvoke $r6.<javaT.securityT.KeyStore.DefaultEntryType$PrivKey1: void <init>(javaT.securityT.KeyStore.DefaultEntryType$1)>(null);
r58 = $r6;
$r7 = new java.security.KeyStore$PrivateKeyEntry;
specialinvoke $r7.<java.security.KeyStore$PrivateKeyEntry: void <init>(java.security.PrivateKey,java.security.cert.Certificate[])>(r58, r57);
r8 = $r7;
r9 = virtualinvoke r8.<java.security.KeyStore$PrivateKeyEntry: java.security.cert.Certificate[] getCertificateChain()>();
$z0 = r9 instanceof java.security.cert.X509Certificate[];
if $z0 == 0 goto $r10 = new java.lang.SecurityException;
";
    "Block 47:
[preds: 46 ] [succs: 48 55 ]
$r23 = new java.security.KeyStore$PrivateKeyEntry;
"->"Block 48:
[preds: 47 ] [succs: 49 55 ]
$r24 = new javaT.securityT.KeyStore.DefaultEntryType$PrivKey1;
";
    "Block 47:
[preds: 46 ] [succs: 48 55 ]
$r23 = new java.security.KeyStore$PrivateKeyEntry;
"->"Block 55:
[preds: 42 43 44 45 46 47 48 49 50 51 52 53 54 ] [succs: 56 57 ]
$r2 := @caughtexception;
r56 = $r2;
$r3 = <java.lang.System: java.io.PrintStream out>;
virtualinvoke $r3.<java.io.PrintStream: void println(java.lang.String)>(\"test 5 passed\");
r57 = newarray (java.security.cert.Certificate)[2];
$r4 = new javaT.securityT.KeyStore.DefaultEntryType$X509Cert;
specialinvoke $r4.<javaT.securityT.KeyStore.DefaultEntryType$X509Cert: void <init>(javaT.securityT.KeyStore.DefaultEntryType$1)>(null);
r57[0] = $r4;
$r5 = new javaT.securityT.KeyStore.DefaultEntryType$X509Cert;
specialinvoke $r5.<javaT.securityT.KeyStore.DefaultEntryType$X509Cert: void <init>(javaT.securityT.KeyStore.DefaultEntryType$1)>(null);
r57[1] = $r5;
$r6 = new javaT.securityT.KeyStore.DefaultEntryType$PrivKey1;
specialinvoke $r6.<javaT.securityT.KeyStore.DefaultEntryType$PrivKey1: void <init>(javaT.securityT.KeyStore.DefaultEntryType$1)>(null);
r58 = $r6;
$r7 = new java.security.KeyStore$PrivateKeyEntry;
specialinvoke $r7.<java.security.KeyStore$PrivateKeyEntry: void <init>(java.security.PrivateKey,java.security.cert.Certificate[])>(r58, r57);
r8 = $r7;
r9 = virtualinvoke r8.<java.security.KeyStore$PrivateKeyEntry: java.security.cert.Certificate[] getCertificateChain()>();
$z0 = r9 instanceof java.security.cert.X509Certificate[];
if $z0 == 0 goto $r10 = new java.lang.SecurityException;
";
    "Block 48:
[preds: 47 ] [succs: 49 55 ]
$r24 = new javaT.securityT.KeyStore.DefaultEntryType$PrivKey1;
"->"Block 49:
[preds: 48 ] [succs: 50 55 ]
specialinvoke $r24.<javaT.securityT.KeyStore.DefaultEntryType$PrivKey1: void <init>(javaT.securityT.KeyStore.DefaultEntryType$1)>(null);
";
    "Block 48:
[preds: 47 ] [succs: 49 55 ]
$r24 = new javaT.securityT.KeyStore.DefaultEntryType$PrivKey1;
"->"Block 55:
[preds: 42 43 44 45 46 47 48 49 50 51 52 53 54 ] [succs: 56 57 ]
$r2 := @caughtexception;
r56 = $r2;
$r3 = <java.lang.System: java.io.PrintStream out>;
virtualinvoke $r3.<java.io.PrintStream: void println(java.lang.String)>(\"test 5 passed\");
r57 = newarray (java.security.cert.Certificate)[2];
$r4 = new javaT.securityT.KeyStore.DefaultEntryType$X509Cert;
specialinvoke $r4.<javaT.securityT.KeyStore.DefaultEntryType$X509Cert: void <init>(javaT.securityT.KeyStore.DefaultEntryType$1)>(null);
r57[0] = $r4;
$r5 = new javaT.securityT.KeyStore.DefaultEntryType$X509Cert;
specialinvoke $r5.<javaT.securityT.KeyStore.DefaultEntryType$X509Cert: void <init>(javaT.securityT.KeyStore.DefaultEntryType$1)>(null);
r57[1] = $r5;
$r6 = new javaT.securityT.KeyStore.DefaultEntryType$PrivKey1;
specialinvoke $r6.<javaT.securityT.KeyStore.DefaultEntryType$PrivKey1: void <init>(javaT.securityT.KeyStore.DefaultEntryType$1)>(null);
r58 = $r6;
$r7 = new java.security.KeyStore$PrivateKeyEntry;
specialinvoke $r7.<java.security.KeyStore$PrivateKeyEntry: void <init>(java.security.PrivateKey,java.security.cert.Certificate[])>(r58, r57);
r8 = $r7;
r9 = virtualinvoke r8.<java.security.KeyStore$PrivateKeyEntry: java.security.cert.Certificate[] getCertificateChain()>();
$z0 = r9 instanceof java.security.cert.X509Certificate[];
if $z0 == 0 goto $r10 = new java.lang.SecurityException;
";
    "Block 49:
[preds: 48 ] [succs: 50 55 ]
specialinvoke $r24.<javaT.securityT.KeyStore.DefaultEntryType$PrivKey1: void <init>(javaT.securityT.KeyStore.DefaultEntryType$1)>(null);
"->"Block 50:
[preds: 49 ] [succs: 51 55 ]
specialinvoke $r23.<java.security.KeyStore$PrivateKeyEntry: void <init>(java.security.PrivateKey,java.security.cert.Certificate[])>($r24, r54);
";
    "Block 49:
[preds: 48 ] [succs: 50 55 ]
specialinvoke $r24.<javaT.securityT.KeyStore.DefaultEntryType$PrivKey1: void <init>(javaT.securityT.KeyStore.DefaultEntryType$1)>(null);
"->"Block 55:
[preds: 42 43 44 45 46 47 48 49 50 51 52 53 54 ] [succs: 56 57 ]
$r2 := @caughtexception;
r56 = $r2;
$r3 = <java.lang.System: java.io.PrintStream out>;
virtualinvoke $r3.<java.io.PrintStream: void println(java.lang.String)>(\"test 5 passed\");
r57 = newarray (java.security.cert.Certificate)[2];
$r4 = new javaT.securityT.KeyStore.DefaultEntryType$X509Cert;
specialinvoke $r4.<javaT.securityT.KeyStore.DefaultEntryType$X509Cert: void <init>(javaT.securityT.KeyStore.DefaultEntryType$1)>(null);
r57[0] = $r4;
$r5 = new javaT.securityT.KeyStore.DefaultEntryType$X509Cert;
specialinvoke $r5.<javaT.securityT.KeyStore.DefaultEntryType$X509Cert: void <init>(javaT.securityT.KeyStore.DefaultEntryType$1)>(null);
r57[1] = $r5;
$r6 = new javaT.securityT.KeyStore.DefaultEntryType$PrivKey1;
specialinvoke $r6.<javaT.securityT.KeyStore.DefaultEntryType$PrivKey1: void <init>(javaT.securityT.KeyStore.DefaultEntryType$1)>(null);
r58 = $r6;
$r7 = new java.security.KeyStore$PrivateKeyEntry;
specialinvoke $r7.<java.security.KeyStore$PrivateKeyEntry: void <init>(java.security.PrivateKey,java.security.cert.Certificate[])>(r58, r57);
r8 = $r7;
r9 = virtualinvoke r8.<java.security.KeyStore$PrivateKeyEntry: java.security.cert.Certificate[] getCertificateChain()>();
$z0 = r9 instanceof java.security.cert.X509Certificate[];
if $z0 == 0 goto $r10 = new java.lang.SecurityException;
";
    "Block 50:
[preds: 49 ] [succs: 51 55 ]
specialinvoke $r23.<java.security.KeyStore$PrivateKeyEntry: void <init>(java.security.PrivateKey,java.security.cert.Certificate[])>($r24, r54);
"->"Block 51:
[preds: 50 ] [succs: 52 55 ]
r55 = $r23;
";
    "Block 50:
[preds: 49 ] [succs: 51 55 ]
specialinvoke $r23.<java.security.KeyStore$PrivateKeyEntry: void <init>(java.security.PrivateKey,java.security.cert.Certificate[])>($r24, r54);
"->"Block 55:
[preds: 42 43 44 45 46 47 48 49 50 51 52 53 54 ] [succs: 56 57 ]
$r2 := @caughtexception;
r56 = $r2;
$r3 = <java.lang.System: java.io.PrintStream out>;
virtualinvoke $r3.<java.io.PrintStream: void println(java.lang.String)>(\"test 5 passed\");
r57 = newarray (java.security.cert.Certificate)[2];
$r4 = new javaT.securityT.KeyStore.DefaultEntryType$X509Cert;
specialinvoke $r4.<javaT.securityT.KeyStore.DefaultEntryType$X509Cert: void <init>(javaT.securityT.KeyStore.DefaultEntryType$1)>(null);
r57[0] = $r4;
$r5 = new javaT.securityT.KeyStore.DefaultEntryType$X509Cert;
specialinvoke $r5.<javaT.securityT.KeyStore.DefaultEntryType$X509Cert: void <init>(javaT.securityT.KeyStore.DefaultEntryType$1)>(null);
r57[1] = $r5;
$r6 = new javaT.securityT.KeyStore.DefaultEntryType$PrivKey1;
specialinvoke $r6.<javaT.securityT.KeyStore.DefaultEntryType$PrivKey1: void <init>(javaT.securityT.KeyStore.DefaultEntryType$1)>(null);
r58 = $r6;
$r7 = new java.security.KeyStore$PrivateKeyEntry;
specialinvoke $r7.<java.security.KeyStore$PrivateKeyEntry: void <init>(java.security.PrivateKey,java.security.cert.Certificate[])>(r58, r57);
r8 = $r7;
r9 = virtualinvoke r8.<java.security.KeyStore$PrivateKeyEntry: java.security.cert.Certificate[] getCertificateChain()>();
$z0 = r9 instanceof java.security.cert.X509Certificate[];
if $z0 == 0 goto $r10 = new java.lang.SecurityException;
";
    "Block 51:
[preds: 50 ] [succs: 52 55 ]
r55 = $r23;
"->"Block 52:
[preds: 51 ] [succs: 53 55 ]
$r25 = new java.lang.SecurityException;
";
    "Block 51:
[preds: 50 ] [succs: 52 55 ]
r55 = $r23;
"->"Block 55:
[preds: 42 43 44 45 46 47 48 49 50 51 52 53 54 ] [succs: 56 57 ]
$r2 := @caughtexception;
r56 = $r2;
$r3 = <java.lang.System: java.io.PrintStream out>;
virtualinvoke $r3.<java.io.PrintStream: void println(java.lang.String)>(\"test 5 passed\");
r57 = newarray (java.security.cert.Certificate)[2];
$r4 = new javaT.securityT.KeyStore.DefaultEntryType$X509Cert;
specialinvoke $r4.<javaT.securityT.KeyStore.DefaultEntryType$X509Cert: void <init>(javaT.securityT.KeyStore.DefaultEntryType$1)>(null);
r57[0] = $r4;
$r5 = new javaT.securityT.KeyStore.DefaultEntryType$X509Cert;
specialinvoke $r5.<javaT.securityT.KeyStore.DefaultEntryType$X509Cert: void <init>(javaT.securityT.KeyStore.DefaultEntryType$1)>(null);
r57[1] = $r5;
$r6 = new javaT.securityT.KeyStore.DefaultEntryType$PrivKey1;
specialinvoke $r6.<javaT.securityT.KeyStore.DefaultEntryType$PrivKey1: void <init>(javaT.securityT.KeyStore.DefaultEntryType$1)>(null);
r58 = $r6;
$r7 = new java.security.KeyStore$PrivateKeyEntry;
specialinvoke $r7.<java.security.KeyStore$PrivateKeyEntry: void <init>(java.security.PrivateKey,java.security.cert.Certificate[])>(r58, r57);
r8 = $r7;
r9 = virtualinvoke r8.<java.security.KeyStore$PrivateKeyEntry: java.security.cert.Certificate[] getCertificateChain()>();
$z0 = r9 instanceof java.security.cert.X509Certificate[];
if $z0 == 0 goto $r10 = new java.lang.SecurityException;
";
    "Block 52:
[preds: 51 ] [succs: 53 55 ]
$r25 = new java.lang.SecurityException;
"->"Block 53:
[preds: 52 ] [succs: 54 55 ]
specialinvoke $r25.<java.lang.SecurityException: void <init>(java.lang.String)>(\"test 5 failed\");
";
    "Block 52:
[preds: 51 ] [succs: 53 55 ]
$r25 = new java.lang.SecurityException;
"->"Block 55:
[preds: 42 43 44 45 46 47 48 49 50 51 52 53 54 ] [succs: 56 57 ]
$r2 := @caughtexception;
r56 = $r2;
$r3 = <java.lang.System: java.io.PrintStream out>;
virtualinvoke $r3.<java.io.PrintStream: void println(java.lang.String)>(\"test 5 passed\");
r57 = newarray (java.security.cert.Certificate)[2];
$r4 = new javaT.securityT.KeyStore.DefaultEntryType$X509Cert;
specialinvoke $r4.<javaT.securityT.KeyStore.DefaultEntryType$X509Cert: void <init>(javaT.securityT.KeyStore.DefaultEntryType$1)>(null);
r57[0] = $r4;
$r5 = new javaT.securityT.KeyStore.DefaultEntryType$X509Cert;
specialinvoke $r5.<javaT.securityT.KeyStore.DefaultEntryType$X509Cert: void <init>(javaT.securityT.KeyStore.DefaultEntryType$1)>(null);
r57[1] = $r5;
$r6 = new javaT.securityT.KeyStore.DefaultEntryType$PrivKey1;
specialinvoke $r6.<javaT.securityT.KeyStore.DefaultEntryType$PrivKey1: void <init>(javaT.securityT.KeyStore.DefaultEntryType$1)>(null);
r58 = $r6;
$r7 = new java.security.KeyStore$PrivateKeyEntry;
specialinvoke $r7.<java.security.KeyStore$PrivateKeyEntry: void <init>(java.security.PrivateKey,java.security.cert.Certificate[])>(r58, r57);
r8 = $r7;
r9 = virtualinvoke r8.<java.security.KeyStore$PrivateKeyEntry: java.security.cert.Certificate[] getCertificateChain()>();
$z0 = r9 instanceof java.security.cert.X509Certificate[];
if $z0 == 0 goto $r10 = new java.lang.SecurityException;
";
    "Block 53:
[preds: 52 ] [succs: 54 55 ]
specialinvoke $r25.<java.lang.SecurityException: void <init>(java.lang.String)>(\"test 5 failed\");
"->"Block 54:
[preds: 53 ] [succs: 55 ]
throw $r25;
";
    "Block 53:
[preds: 52 ] [succs: 54 55 ]
specialinvoke $r25.<java.lang.SecurityException: void <init>(java.lang.String)>(\"test 5 failed\");
"->"Block 55:
[preds: 42 43 44 45 46 47 48 49 50 51 52 53 54 ] [succs: 56 57 ]
$r2 := @caughtexception;
r56 = $r2;
$r3 = <java.lang.System: java.io.PrintStream out>;
virtualinvoke $r3.<java.io.PrintStream: void println(java.lang.String)>(\"test 5 passed\");
r57 = newarray (java.security.cert.Certificate)[2];
$r4 = new javaT.securityT.KeyStore.DefaultEntryType$X509Cert;
specialinvoke $r4.<javaT.securityT.KeyStore.DefaultEntryType$X509Cert: void <init>(javaT.securityT.KeyStore.DefaultEntryType$1)>(null);
r57[0] = $r4;
$r5 = new javaT.securityT.KeyStore.DefaultEntryType$X509Cert;
specialinvoke $r5.<javaT.securityT.KeyStore.DefaultEntryType$X509Cert: void <init>(javaT.securityT.KeyStore.DefaultEntryType$1)>(null);
r57[1] = $r5;
$r6 = new javaT.securityT.KeyStore.DefaultEntryType$PrivKey1;
specialinvoke $r6.<javaT.securityT.KeyStore.DefaultEntryType$PrivKey1: void <init>(javaT.securityT.KeyStore.DefaultEntryType$1)>(null);
r58 = $r6;
$r7 = new java.security.KeyStore$PrivateKeyEntry;
specialinvoke $r7.<java.security.KeyStore$PrivateKeyEntry: void <init>(java.security.PrivateKey,java.security.cert.Certificate[])>(r58, r57);
r8 = $r7;
r9 = virtualinvoke r8.<java.security.KeyStore$PrivateKeyEntry: java.security.cert.Certificate[] getCertificateChain()>();
$z0 = r9 instanceof java.security.cert.X509Certificate[];
if $z0 == 0 goto $r10 = new java.lang.SecurityException;
";
    "Block 54:
[preds: 53 ] [succs: 55 ]
throw $r25;
"->"Block 55:
[preds: 42 43 44 45 46 47 48 49 50 51 52 53 54 ] [succs: 56 57 ]
$r2 := @caughtexception;
r56 = $r2;
$r3 = <java.lang.System: java.io.PrintStream out>;
virtualinvoke $r3.<java.io.PrintStream: void println(java.lang.String)>(\"test 5 passed\");
r57 = newarray (java.security.cert.Certificate)[2];
$r4 = new javaT.securityT.KeyStore.DefaultEntryType$X509Cert;
specialinvoke $r4.<javaT.securityT.KeyStore.DefaultEntryType$X509Cert: void <init>(javaT.securityT.KeyStore.DefaultEntryType$1)>(null);
r57[0] = $r4;
$r5 = new javaT.securityT.KeyStore.DefaultEntryType$X509Cert;
specialinvoke $r5.<javaT.securityT.KeyStore.DefaultEntryType$X509Cert: void <init>(javaT.securityT.KeyStore.DefaultEntryType$1)>(null);
r57[1] = $r5;
$r6 = new javaT.securityT.KeyStore.DefaultEntryType$PrivKey1;
specialinvoke $r6.<javaT.securityT.KeyStore.DefaultEntryType$PrivKey1: void <init>(javaT.securityT.KeyStore.DefaultEntryType$1)>(null);
r58 = $r6;
$r7 = new java.security.KeyStore$PrivateKeyEntry;
specialinvoke $r7.<java.security.KeyStore$PrivateKeyEntry: void <init>(java.security.PrivateKey,java.security.cert.Certificate[])>(r58, r57);
r8 = $r7;
r9 = virtualinvoke r8.<java.security.KeyStore$PrivateKeyEntry: java.security.cert.Certificate[] getCertificateChain()>();
$z0 = r9 instanceof java.security.cert.X509Certificate[];
if $z0 == 0 goto $r10 = new java.lang.SecurityException;
";
    "Block 55:
[preds: 42 43 44 45 46 47 48 49 50 51 52 53 54 ] [succs: 56 57 ]
$r2 := @caughtexception;
r56 = $r2;
$r3 = <java.lang.System: java.io.PrintStream out>;
virtualinvoke $r3.<java.io.PrintStream: void println(java.lang.String)>(\"test 5 passed\");
r57 = newarray (java.security.cert.Certificate)[2];
$r4 = new javaT.securityT.KeyStore.DefaultEntryType$X509Cert;
specialinvoke $r4.<javaT.securityT.KeyStore.DefaultEntryType$X509Cert: void <init>(javaT.securityT.KeyStore.DefaultEntryType$1)>(null);
r57[0] = $r4;
$r5 = new javaT.securityT.KeyStore.DefaultEntryType$X509Cert;
specialinvoke $r5.<javaT.securityT.KeyStore.DefaultEntryType$X509Cert: void <init>(javaT.securityT.KeyStore.DefaultEntryType$1)>(null);
r57[1] = $r5;
$r6 = new javaT.securityT.KeyStore.DefaultEntryType$PrivKey1;
specialinvoke $r6.<javaT.securityT.KeyStore.DefaultEntryType$PrivKey1: void <init>(javaT.securityT.KeyStore.DefaultEntryType$1)>(null);
r58 = $r6;
$r7 = new java.security.KeyStore$PrivateKeyEntry;
specialinvoke $r7.<java.security.KeyStore$PrivateKeyEntry: void <init>(java.security.PrivateKey,java.security.cert.Certificate[])>(r58, r57);
r8 = $r7;
r9 = virtualinvoke r8.<java.security.KeyStore$PrivateKeyEntry: java.security.cert.Certificate[] getCertificateChain()>();
$z0 = r9 instanceof java.security.cert.X509Certificate[];
if $z0 == 0 goto $r10 = new java.lang.SecurityException;
"->"Block 56:
[preds: 55 ] [succs: 58 ]
$r11 = <java.lang.System: java.io.PrintStream out>;
virtualinvoke $r11.<java.io.PrintStream: void println(java.lang.String)>(\"test 6 passed\");
goto [?= $i0 = lengthof r9];
";
    "Block 55:
[preds: 42 43 44 45 46 47 48 49 50 51 52 53 54 ] [succs: 56 57 ]
$r2 := @caughtexception;
r56 = $r2;
$r3 = <java.lang.System: java.io.PrintStream out>;
virtualinvoke $r3.<java.io.PrintStream: void println(java.lang.String)>(\"test 5 passed\");
r57 = newarray (java.security.cert.Certificate)[2];
$r4 = new javaT.securityT.KeyStore.DefaultEntryType$X509Cert;
specialinvoke $r4.<javaT.securityT.KeyStore.DefaultEntryType$X509Cert: void <init>(javaT.securityT.KeyStore.DefaultEntryType$1)>(null);
r57[0] = $r4;
$r5 = new javaT.securityT.KeyStore.DefaultEntryType$X509Cert;
specialinvoke $r5.<javaT.securityT.KeyStore.DefaultEntryType$X509Cert: void <init>(javaT.securityT.KeyStore.DefaultEntryType$1)>(null);
r57[1] = $r5;
$r6 = new javaT.securityT.KeyStore.DefaultEntryType$PrivKey1;
specialinvoke $r6.<javaT.securityT.KeyStore.DefaultEntryType$PrivKey1: void <init>(javaT.securityT.KeyStore.DefaultEntryType$1)>(null);
r58 = $r6;
$r7 = new java.security.KeyStore$PrivateKeyEntry;
specialinvoke $r7.<java.security.KeyStore$PrivateKeyEntry: void <init>(java.security.PrivateKey,java.security.cert.Certificate[])>(r58, r57);
r8 = $r7;
r9 = virtualinvoke r8.<java.security.KeyStore$PrivateKeyEntry: java.security.cert.Certificate[] getCertificateChain()>();
$z0 = r9 instanceof java.security.cert.X509Certificate[];
if $z0 == 0 goto $r10 = new java.lang.SecurityException;
"->"Block 57:
[preds: 55 ] [succs: ]
$r10 = new java.lang.SecurityException;
specialinvoke $r10.<java.lang.SecurityException: void <init>(java.lang.String)>(\"test 6 failed\");
throw $r10;
";
    "Block 56:
[preds: 55 ] [succs: 58 ]
$r11 = <java.lang.System: java.io.PrintStream out>;
virtualinvoke $r11.<java.io.PrintStream: void println(java.lang.String)>(\"test 6 passed\");
goto [?= $i0 = lengthof r9];
"->"Block 58:
[preds: 56 ] [succs: 59 62 ]
$i0 = lengthof r9;
if $i0 != 2 goto $r59 = new java.lang.SecurityException;
";
    "Block 58:
[preds: 56 ] [succs: 59 62 ]
$i0 = lengthof r9;
if $i0 != 2 goto $r59 = new java.lang.SecurityException;
"->"Block 59:
[preds: 58 ] [succs: 60 62 ]
$r13 = r9[0];
$r12 = r57[0];
if $r13 != $r12 goto $r59 = new java.lang.SecurityException;
";
    "Block 58:
[preds: 56 ] [succs: 59 62 ]
$i0 = lengthof r9;
if $i0 != 2 goto $r59 = new java.lang.SecurityException;
"->"Block 62:
[preds: 58 59 60 ] [succs: ]
$r59 = new java.lang.SecurityException;
specialinvoke $r59.<java.lang.SecurityException: void <init>(java.lang.String)>(\"test 7 failed\");
throw $r59;
";
    "Block 59:
[preds: 58 ] [succs: 60 62 ]
$r13 = r9[0];
$r12 = r57[0];
if $r13 != $r12 goto $r59 = new java.lang.SecurityException;
"->"Block 60:
[preds: 59 ] [succs: 61 62 ]
$r15 = r9[1];
$r14 = r57[1];
if $r15 != $r14 goto $r59 = new java.lang.SecurityException;
";
    "Block 59:
[preds: 58 ] [succs: 60 62 ]
$r13 = r9[0];
$r12 = r57[0];
if $r13 != $r12 goto $r59 = new java.lang.SecurityException;
"->"Block 62:
[preds: 58 59 60 ] [succs: ]
$r59 = new java.lang.SecurityException;
specialinvoke $r59.<java.lang.SecurityException: void <init>(java.lang.String)>(\"test 7 failed\");
throw $r59;
";
    "Block 60:
[preds: 59 ] [succs: 61 62 ]
$r15 = r9[1];
$r14 = r57[1];
if $r15 != $r14 goto $r59 = new java.lang.SecurityException;
"->"Block 61:
[preds: 60 ] [succs: 63 ]
$r16 = <java.lang.System: java.io.PrintStream out>;
virtualinvoke $r16.<java.io.PrintStream: void println(java.lang.String)>(\"test 7 passed\");
goto [?= $r17 = virtualinvoke r8.<java.security.KeyStore$PrivateKeyEntry: java.security.PrivateKey getPrivateKey()>()];
";
    "Block 60:
[preds: 59 ] [succs: 61 62 ]
$r15 = r9[1];
$r14 = r57[1];
if $r15 != $r14 goto $r59 = new java.lang.SecurityException;
"->"Block 62:
[preds: 58 59 60 ] [succs: ]
$r59 = new java.lang.SecurityException;
specialinvoke $r59.<java.lang.SecurityException: void <init>(java.lang.String)>(\"test 7 failed\");
throw $r59;
";
    "Block 61:
[preds: 60 ] [succs: 63 ]
$r16 = <java.lang.System: java.io.PrintStream out>;
virtualinvoke $r16.<java.io.PrintStream: void println(java.lang.String)>(\"test 7 passed\");
goto [?= $r17 = virtualinvoke r8.<java.security.KeyStore$PrivateKeyEntry: java.security.PrivateKey getPrivateKey()>()];
"->"Block 63:
[preds: 61 ] [succs: 64 65 ]
$r17 = virtualinvoke r8.<java.security.KeyStore$PrivateKeyEntry: java.security.PrivateKey getPrivateKey()>();
if $r17 != r58 goto $r18 = new java.lang.SecurityException;
";
    "Block 63:
[preds: 61 ] [succs: 64 65 ]
$r17 = virtualinvoke r8.<java.security.KeyStore$PrivateKeyEntry: java.security.PrivateKey getPrivateKey()>();
if $r17 != r58 goto $r18 = new java.lang.SecurityException;
"->"Block 64:
[preds: 63 ] [succs: 66 ]
$r19 = <java.lang.System: java.io.PrintStream out>;
virtualinvoke $r19.<java.io.PrintStream: void println(java.lang.String)>(\"test 8 passed\");
goto [?= return];
";
    "Block 63:
[preds: 61 ] [succs: 64 65 ]
$r17 = virtualinvoke r8.<java.security.KeyStore$PrivateKeyEntry: java.security.PrivateKey getPrivateKey()>();
if $r17 != r58 goto $r18 = new java.lang.SecurityException;
"->"Block 65:
[preds: 63 ] [succs: ]
$r18 = new java.lang.SecurityException;
specialinvoke $r18.<java.lang.SecurityException: void <init>(java.lang.String)>(\"test 8 failed\");
throw $r18;
";
    "Block 64:
[preds: 63 ] [succs: 66 ]
$r19 = <java.lang.System: java.io.PrintStream out>;
virtualinvoke $r19.<java.io.PrintStream: void println(java.lang.String)>(\"test 8 passed\");
goto [?= return];
"->"Block 66:
[preds: 64 ] [succs: ]
return;
";
}
